pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -203,12 +203,14 @@ def test_step(self, *args, **kwargs):
     def predict(self, *args, **kwargs):
         return self.lightning_module.predict(*args, **kwargs)
 
-    def save_checkpoint(self, checkpoint: Dict[str, Any], filepath: str) -> None:
+    def save_checkpoint(self, filepath: str, weights_only: bool = False) -> None:
```

Comment:
@tchaton @kaushikb11 where/when we changed this API?

Reply:
Changed in this PR: https://github.com/PyTorchLightning/pytorch-lightning/commit/646cf2f7d4876b5bf53b39960b2b82b722767ada
You can use git blame to find the associated PR.

Reply:
yes I know I just say that the API is different / not back compatible